packages/hdwallet-ledger/src/ledger.ts

```diff
@@ -331,6 +333,7 @@ export class LedgerHDWallet
   readonly _supportsBase = true;
   readonly _supportsThorchain = true;
   readonly _supportsCosmos = true;
+  readonly _supportsSolana = true;
```

Comment:
implements core.SolanaWallet for LedgerHDWallet

Reply:
170b4ee2bc0222ffeb4517369ee02a1c7904feca